bindata/assets/routes/console-route.yaml

```diff
@@ -7,6 +7,10 @@ metadata:
   namespace: openshift-console
   annotations:
     haproxy.router.openshift.io/timeout: 5m
+    haproxy.router.openshift.io/rate-limit-connections: "true"
+    haproxy.router.openshift.io/rate-limit-connections.concurrent-tcp: 3
```

Comment:
Will this cause problems for console itself? Console frequently holds more watches than this open, and we make many concurrent requests on load in the background for API discovery. This will limit it to 3 only from a browser, right? Does this setting impact websockets?

Reply:
> Will this cause problems for console itself? Console frequently holds more watches than this open, and we make many concurrent requests on load in the background for API discovery. This will limit it to 3 only from a browser, right? Does this setting impact websockets?

@spadgett Right, 3 requests per second.
cc: @jhadvig